src/components/BuySellForm/BuySellAmount/BuySellAmount.tsx

```diff
@@ -51,6 +51,7 @@ const BuySellAmount = ({
         setBuySellAmount(
             FormatUtils.formatMoney(Number(inputValue) * Number(calculatedRate), {
                 currency: localCurrency,
+                decimalPlaces: 2,
```

Comment:
Explained this in the description for this PR, but for some reason, the default decimalPlaces for formatMoney is not being applied to this.